.github/workflows/on_push.yaml

```diff
@@ -0,0 +1,10 @@
+on:
+  push:
+    branches:
+      - '*.*/edge'
```

Comment:
This wont run on main.  I am not sure how you want to structure your branches, but you may want to change this.

Reply:
Been discussing this with @taurus-forever . It is expected that it won't work on main but only on "version/edge", the branches that should be publishing to dockerhub